CHANGELOG.rst

```diff
@@ -3,6 +3,7 @@ Changelog
 
 master
 ------
+- (`#52 <https://github.com/znicholls/silicone/pull/52>`_) Made the Lead Gas infiller use the average latest data. Updated infill_composite_values to work with the latest data.
```

Comment:
```suggestion
- (`#52 <https://github.com/znicholls/silicone/pull/52>`_) Made the Lead Gas infiller use the average latest data rather than being restricted to a single value. Updated infill_composite_values to work with the latest data.
```